src/main/java/RacingGame/model/CarsManager.java

```diff
@@ -2,49 +2,45 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import java.util.stream.Collectors;
+import java.util.stream.IntStream;
 
 public class CarsManager {
+    private List<Car> cars;
 
-    private List<Car> cars = new ArrayList<>();
+    public CarsManager() {
+        this.cars = new ArrayList<>();
+    }
 
     public CarsManager(List<Car> cars) {
         this.cars = cars;
     }
 
     public static CarsManager withCarCount(int carCount) {
-        List<Car> cars = new ArrayList<>();
-        for (int i = 0; i < carCount; i++) {
-            cars.add(new Car());
-        }
+        List<Car> cars = IntStream.range(0, carCount)
+                .mapToObj(i -> new Car())
+                .collect(Collectors.toList());
 
         return new CarsManager(cars);
     }
 
     public static CarsManager withCarNames(List<String> carNames) {
-        List<Car> cars = new ArrayList<>();
-
-        for (String carName : carNames) {
-            cars.add(new Car(carName));
-        }
+        List<Car> cars = carNames.stream()
+                .map(carname -> new Car(carname))
```

Comment:
```suggestion
                .map(Car::new)
```
메서드 레퍼런스를 활용할 수 있어보입니다!